tests/network-tests/src/flows/proposals/cancellingProposal.ts

```diff
@@ -20,11 +21,11 @@ export default async function cancellingProposals({ api, query, lock }: FlowProp
   const createProposalFixture = new CreateProposalsFixture(api, query, [
     {
       type: 'Signal',
-      details: 'Proposal to cancel',
+      details: ('Proposal to cancel' as unknown) as Bytes,
```

Comment:
Use `createType('Bytes', 'Proposal to cancel')`, always try to avoid type assertions.